src/main/resources/org/jenkinsci/plugins/configfiles/folder/FolderConfigFileAction/configfiles.jelly

```diff
@@ -24,10 +24,9 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
-		 xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:cf="/lib/configfiles">
+    xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:cf="/lib/configfiles">
 	<l:layout permission="${it.CONFIGURE}" norefresh="true">
 
-
 		<cf:sitepanel />
 
 		<l:main-panel>
```

Comment:
I am not sure why `intro` is being exported via `j:out` below. Smells like a bug.